src/headers.ts

```diff
@@ -855,12 +1019,38 @@ export const inchi_OutputStruct = NAPIStructType({
   WarningFlags: NAPIArrayType(NAPIArrayType(refNAPI.types.ulong, 2), 2),
 });
 export interface INCHIOutputStruct {
+  /**
+   * Array of INCHIAtom elements
+   */
   atom: INCHIAtom;
+  /**
+   * Array of INCHIStereo0D 0D stereo elements or NULL
+   */
   stereo0D: INCHIStereo0D;
+  /**
+   * number of atoms in the structure
+   */
   numAtoms: number;
+  /**
+   * number of 0D stereo elements
+   */
   numStereo0D: number;
+  /**
+   * Error/warning ASCIIZ message
+   */
   szMessage: string | null;
+  /**
+   * log-file ASCIIZ string, contains a human-readable list of recognized options and possibly an Error/warn message
+   */
   szLog: string | null;
+  /**
+   * warnings
+   * [x][y]:
+   * x=0 => Reconnected if present in InChI otherwise Disconnected/Normal
+   * x=1 => Disconnected layer if Reconnected layer is present
+   * y=0 => Fixed-H layer
+   * y=1 => Main layer or Mobile-H
```

Comment:
```suggestion
   * warnings [x][y]:
   * - `x=0` => Reconnected if present in InChI otherwise Disconnected/Normal
   * - `x=1` => Disconnected layer if Reconnected layer is present
   * - `y=0` => Fixed-H layer
   * - `y=1` => Main layer or Mobile-H
```